### PR TITLE
examples: exit gimbal script

### PR DESCRIPTION
**Pull request by julianoes**
Otherwise it hangs when done.

### CODE REVIEW

File: examples/gimbal.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ async def run():
     await drone.connect(system_address="udp://:14540")
 
     # Start printing gimbal position updates
-    asyncio.ensure_future(print_gimbal_position(drone))
+    print_gimal_position_task = asyncio.ensure_future(print_gimbal_position(drone))
```
Comment (JonasVautherin) [inline examples/gimbal.py:13]:
```suggestion
    print_gimbal_position_task = asyncio.ensure_future(print_gimbal_position(drone))
```

File: examples/gimbal.py [hunk 2]
```diff
@@ -71,6 +71,8 @@ async def run():
     print("Release control of gimbal again")
     await drone.gimbal.release_control()
 
+    print_gimal_position_task.cancel()
```
Comment (JonasVautherin) [inline examples/gimbal.py:74]:
```suggestion
    print_gimbal_position_task.cancel()
```

**Review by JonasVautherin — APPROVED** [review]
Just a typo, otherwise LGTM!

**Comment by julianoes** [comment]
Whops, thanks.

**Pull request merged**